.gitattributes

```diff
@@ -0,0 +1,5 @@
+*.gradle text eol=lf
+*.java text eol=lf
+*.json text eol=lf
+*.md text eol=lf
+*.xml text eol=lf
```

Comment:
New line at end of file

Reply:
No new line in allwpilib

Reply:
Well there should be. Thats an issue there. All files need one.

Reply:
Nvm, I read it wrong sry